src/rust/engine/build.rs

```diff
@@ -31,4 +31,7 @@ fn main() {
   // NB: The native extension only works with the Python interpreter version it was built with
   // (e.g. Python 3.7 vs 3.8).
   println!("cargo:rerun-if-env-changed=PY");
+  if let Ok(py_var) = std::env::var("PY") {
```

Comment:
Seems like killing the line above is the right thing to do, or at least - I'm unclear how this is different from above in a way that its separately useful but not replace worthy.

Reply:
So: my uncertainty was just around what would happen if the variable changed but not the content...? But you're probably right that only the latter line is actually necessary.